androidapp/app/src/main/java/com/example/lanweather/MainActivity.kt

```diff
@@ -54,7 +54,7 @@ class MainActivity : AppCompatActivity() {
         // menu should be considered as top level destinations.
         val appBarConfiguration = AppBarConfiguration(setOf(
             R.id.navigation_home, R.id.navigation_forecast, R.id.navigation_settings))
-//        setupActionBarWithNavController(navController, appBarConfiguration)
+//        setupActionBarWithNavController(this, navController, appBarConfiguration)
```

Comment:
change on a function call in a comment?

Reply:
There are apparently a few different `setupActionBarWithNavController()` methods, and I must've been messing around with those. I reverted it to the original method.